src/Wizard/__mocks__/mockEmbeddedWizardManager.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import helpers from '../../common/helpers'
+import { Button, Wizard } from '../../index'
+import { mockWizardItems } from './mockWizardItems'
+import {
+  renderWizardSteps,
+  renderSidebarItems,
+  renderWizardContents
+} from './mockWizardRenderers'
+
+export class MockEmbeddedWizardManager extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      activeStepIndex: props.initialStepIndex || 0,
+      activeSubStepIndex: props.initialSubStepIndex || 0
+    }
+
+    helpers.bindMethods(this, [
+      'open',
+      'close',
+      'onSidebarItemClick',
+      'onStepClick',
+      'onNextButtonClick',
+      'onBackButtonClick'
+    ])
+    this.open = this.open.bind(this)
+    this.close = this.close.bind(this)
+    this.onSidebarItemClick = this.onSidebarItemClick.bind(this)
+    this.onStepClick = this.onStepClick.bind(this)
+    this.onNextButtonClick = this.onNextButtonClick.bind(this)
+    this.onBackButtonClick = this.onBackButtonClick.bind(this)
+  }
+  open() {
+    this.setState({ showModal: true })
+  }
+  close() {
+    this.setState({ showModal: false })
+  }
+  onSidebarItemClick(stepIndex, subStepIndex) {
+    this.setState({
+      activeStepIndex: stepIndex,
+      activeSubStepIndex: subStepIndex
+    })
+  }
+  onStepClick(stepIndex) {
+    if (stepIndex === this.state.activeStepIndex) {
+      return
+    }
+    this.setState({
+      activeStepIndex: stepIndex,
+      activeSubStepIndex: 0
+    })
+  }
+  onNextButtonClick() {
+    const { steps } = this.props
+    const { activeStepIndex, activeSubStepIndex } = this.state
+    const activeStep = steps[activeStepIndex]
+
+    if (activeSubStepIndex < activeStep.subSteps.length - 1) {
+      this.setState(prevState => ({
+        activeSubStepIndex: prevState.activeSubStepIndex + 1
+      }))
+    } else if (activeStepIndex < steps.length - 1) {
+      this.setState(prevState => ({
+        activeStepIndex: prevState.activeStepIndex + 1,
+        activeSubStepIndex: 0
+      }))
+    }
+  }
+  onBackButtonClick() {
+    const { steps } = this.props
+    const { activeStepIndex, activeSubStepIndex } = this.state
+
+    if (activeSubStepIndex > 0) {
+      this.setState(prevState => ({
+        activeSubStepIndex: prevState.activeSubStepIndex - 1
+      }))
+    } else if (activeStepIndex > 0) {
+      this.setState(prevState => ({
+        activeStepIndex: prevState.activeStepIndex - 1,
+        activeSubStepIndex:
+          steps[prevState.activeStepIndex - 1].subSteps.length - 1
+      }))
+    }
+  }
+  render() {
+    const { activeStepIndex, activeSubStepIndex } = this.state
+
+    return (
+      <Wizard className="wizard-pf">
```

Comment:
Currently, this class `wizard-pf` sets the width of the wizard to 900px wide, which doesn't make sense for the embedded wizard. Is it possible to remove this class for the embedded wizard storybook example?

Reply:
OK - after hearing your's and @waldenraines feedback, I am thinking we can have `<Wizard embedded>` optional boolean prop which tells Wizard what the default class is and feeds down to child components interested. This should get us there...